UI/EncyclopediaDetailPanel.cpp

```diff
@@ -2303,15 +2303,23 @@ namespace {
             detailed_description += "\n\n" + UserString("LIKES");
             int count = 0;
             for (const auto& s : species->Likes())
-                detailed_description += (count++ == 0 ? "" : ",  ") + UserString(s);
+                detailed_description
+                .append(count++ == 0 ? "" : ",  ")
+                //.append(LinkTaggedPresetText(VarText::ENCYCLOPEDIA_TAG,s,UserString(s)))
```

Comment:
It might be a bit slow, but you could make a helper to look up a like/dislike string in various content managers and return whichever type of thing it finds the string in. So check if there is a special with that name, or a policy, or a building, or (whatever). Would be similar to https://github.com/freeorion/freeorion/blob/master/universe/Conditions.cpp#L9945

Reply:
doesnt have to be too fast i think, since it'll only be executed when the player clicks a pedia link, and a couple milliseconds more wouldnt be noticed. i'll try that, thx